eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -25,12 +28,17 @@ class Report:
     One instance of this class is instantiated in the running of the pipeline.
     """
 
-    def __init__(self, unavailable_efo=None):
+    def __init__(self, trait_mappings=None, unavailable_efo=None):
```

Comment:
This (http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments) explains why i didn't put the set and dictionary in the defaults of the Report constructor. There is currently only one report created each time the application runs, but I thought it'd be better practice to still implement it that way.